doc/sphinx/user/methods/advection-stabilization/entropy-viscosity.md

```diff
@@ -47,19 +44,25 @@ approximation is good.
 The above definition assumes the entropy residual exponent
 ("alpha" in the parameter files, see
 {ref}`parameters:Discretization/Stabilization_20parameters/alpha`38]) is set
-to 2 (the default and recommended). For the choice of 1 for
-"alpha," the entropy viscosity is defined as
+to 2 (the default and recommended). For the choice of 1 for "alpha," the
+entropy viscosity is defined as
 ```{math}
 v_h^E \vert_K = \alpha_E \frac{h |\Omega| \cdot \| \mathbf u \|_{\infty,K} \cdot \| r_E \|_{\infty, K}}
  {\| \mathbf u \|_{\infty,\Omega} \cdot (T_\text{max} - T_\text{min})}.
  ```
 instead.
 
 An additional parameter is the strain rate scaling factor "gamma"
-(see {ref}`parameters:Discretization/Stabilization_20parameters/gamma`39]),
+(see {ref}`parameters:Discretization/Stabilization_20parameters/gamma`),
 which changes the definition of the maximum dissipation $\nu_h^\text{max}$ to
 ```{math}
 v_h^\text{max} \vert_K = \alpha_\text{max} h \|\lvert\mathbf u\rvert + \gamma h_K \lvert\varepsilon (\mathbf u)\rvert\|_{\infty,K},
 ```
 where $\gamma\geq 0$ is the aforementioned parameter in front of the strain
 rate.
+
+:::{admonition} TODO
+:class: error
+
+parameter references
```

Comment:
As of now all parameter references don't link to anything. I've been leaving these TODOs in every section that has parameter references so that we can go back and fix things if and when we have something specific to link to

Reply:
Please replace these comments with explanations that also work for someone else. (e.g. what you wrote in your comment). You never know who will fix these things up later (or if you yourself will remember), so it is important to not rely on your current knowledge when writing the comments.